julearn/transformers/confounds.py

```diff
@@ -98,17 +104,17 @@ def fit_confound_models(X):
         return self
 
     def transform(self, X):
-        """Removes confounds from data
+        """Remove confounds from data.
 
         Parameters
         ----------
         X : pandas.DataFrame
```

Comment:
Please adjust the types to pd.DataFrame. (I know it was not you writing pandas, but to make it consistent with the new better style of using pd.DataFrame